packages/react-components/src/atoms/__tests__/Subtitle.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Subtitle from '../Subtitle';
+
+it('renders the text in an <h4>', () => {
+  const { getByText } = render(<Subtitle>text</Subtitle>);
+  expect(getByText('text').tagName).toBe('H5');
+});
+
+it('applies the text margin', () => {
+  const { getByText } = render(<Subtitle>text</Subtitle>);
+  const { marginTop } = getComputedStyle(getByText('text'));
+  expect(marginTop).toMatchInlineSnapshot(`"12px"`);
+});
+
+it('renders the text in regular size bold', () => {
```

Comment:
There is no assertion around bold

Reply:
True dat. I noticed the other heading tests were only checking font size, not weight, and skipped it